packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -236,6 +238,7 @@ async function fetchLatestSnapshotCore(
                         fetchTimeout = undefined;
                     }
                 });
+                const networkTimeJs = performance.now() - start;
```

Comment:
Seems like we need to subtract spRequestDuration from this so that it is calculated same way as networkTime?

Reply:
Also, I think we can calculate this time also and match with fetchStartToResponseEndTime?

Reply:
I do not think so. This measures end-to-end, irrespective of how much of that transferring bytes vs. waiting for SPO.
It's same for networkTimeBrowser.
Maybe you are thinking about responseNetworkTime ? That one measures time from first byte in to last byte in, that can theoretically mean that spRequestDuration was excluded. But if you look at data, it's not clear that's actually the case. So I'd not make any conclusions in this code, but rather report numbers and use queries to analyze it.

Reply:
If you look at this:
```suggestion
 if (spReqDuration !== undefined && networkTimeBrowser !== undefined) {
                            networkTimeBrowser = networkTimeBrowser - parseInt(spReqDuration, 10);
                        }
                        
```
We do subtract 6the server time.